centaur/src/main/resources/standardTestCases/wdl_draft3/callable_imports/my_workflow.wdl

```diff
@@ -0,0 +1,10 @@
+version draft-3
+
+workflow my_workflow {
+  input{
```

Comment:
[optional / pedantic] missing space messes up github highlighting (or maybe github highlighting is what needs an update?)

Reply:
A little bit of both I think but github highlighting is based on our sublime syntax - https://github.com/broadinstitute/wdl-sublime-syntax-highlighter - and probably needs an update (it also needs an update for draft-3 now that I think of it...)